lib/rnrs/base.sld

```diff
@@ -0,0 +1,231 @@
+          cons
+          cos
+          define
+          define-syntax
+          denominator
+          div
+          div-and-mod
+          div0
+          div0-and-mod0
+          dynamic-wind
+          else
+          eq?
+          equal?
+          eqv?
+          error
+          even?
+          exact
+          exact-integer-sqrt
+          exact?
+          exp
+          expt
+          finite?
+          floor
+          for-each
+          gcd
+          identifier-syntax
+          if
+          imag-part
+          inexact
+          inexact?
+          infinite?
+          integer->char
+          integer-valued?
+          integer?
+          lambda
+          lcm
+          length
+          let
+          let*
+          let*-values
+          let-syntax
+          let-values
+          letrec
+          letrec*
+          letrec-syntax
+          list
+          list->string
+          list->vector
+          list-ref
+          list-tail
+          list?
+          log
+          magnitude
+          make-polar
+          make-rectangular
+          make-string
+          make-vector
+          map
+          max
+          min
+          mod
+          mod0
+          nan?
+          negative?
+          not
+          null?
+          number->string
+          number?
+          numerator
+          odd?
+          or
+          pair?
+          positive?
+          procedure?
+          quasiquote
+          quote
+          rational-valued?
+          rational?
+          rationalize
+          real-part
+          real-valued?
+          real?
+          reverse
+          round
+          set!
+          sin
+          sqrt
+          string
+          string->list
+          string->number
+          string->symbol
+          string-append
+          string-copy
+          string-for-each
+          string-length
+          string-ref
+          string<=?
+          string<?
+          string=?
+          string>=?
+          string>?
+          string?
+          substring
+          symbol->string
+          symbol=?
+          symbol?
+          syntax-rules
+          tan
+          truncate
+          unquote
+          unquote-splicing
+          values
+          vector
+          vector->list
+          vector-fill!
+          vector-for-each
+          vector-length
+          vector-map
+          vector-ref
+          vector-set!
+          vector?
+          zero?)
+  (import (rename (scheme base)
+                  (error r7rs:error))
+          (scheme cxr)
+          (scheme inexact)
+          (scheme complex)
+          (rename (srfi 141)
+                  (euclidean-quotient div)
+                  (euclidean-remainder mod)
+                  (euclidean/ div-and-mod)
+                  (balanced-quotient div0)
+                  (balanced-remainder mod0)
+                  (balanced/ div0-and-mod0))
+          (except (chibi ast) error)
+          (chibi show))
+
+  (define-syntax assert
```

Comment:
Is `(chibi assert)` a proper superset of R6RS assert?  It gives better reporting.